pkg/cmd/run.go

```diff
@@ -801,6 +800,8 @@ func resolvePodTemplate(ctx context.Context, templateSrc string, spec *v1.Integr
 func configureTraits(options []string, catalog *trait.Catalog) (map[string]v1.TraitSpec, error) {
 	traits := make(map[string]map[string]interface{})
 
+	// remove any duplicate trait parameters in case it is set as modeline or command line parameters
+	util.StringsSliceRemoveDuplicates(&options)
```

Comment:
What is the precedence between modeline and CLI options? I think the CLI options should override duplicated ones from modeline?

Reply:
At this point there is no difference about the trait values, so the duplicate routine checks the trait name and value. I think it would be appropriate to check only the names, similar to the property key names, wdyt ?

Reply:
Now the code checks for trait names only and the cli traits has precedence.